### DISCUSSION TITLE
Multiple plex server

### DISCUSSION THREAD

**Discussion by lonelyb3ar** [sct/overseerr#2664] [Feature Requests]
Will yous be looking at adding support for multiple plex servers ?

**Comment by TheCatLady** [comment]
No, each Overseerr instance can only be connected to a single Plex server. There are no plans to change this.

**Comment by VikingPlex** [comment]
Are there still no plans to change this, I would love to be able to add users into request system from multiple servers but all in the same place ? What is getting in the way of adding features for multiple plex accounts ?
